### DISCUSSION TITLE
forge  snapshot -m testWithdrawFromMultipleFunders

### DISCUSSION THREAD

**Discussion by zubyoz** [NomicFoundation/hardhat#5697] [General]
Hello everyone, I'm having touble figuring this out, please has anyone experienced this problem below, I am learning the Foundry Fundamentals (calculate withdraw gas cost)

forge  snapshot -m testWithdrawFromMultipleFunders
error: unexpected argument '-m' found

Usage: forge snapshot [OPTIONS]

For more information, try '--help'.

**Comment by kanej** [comment]
This is the Hardhat project, `forge` is part of the Foundry project. I would ask on their github: https://github.com/foundry-rs/foundry
